packages/aws-cdk-lib/aws-cloudwatch/lib/alarm.ts

```diff
@@ -60,6 +61,16 @@ export interface AnomalyDetectionAlarmProps extends CreateAlarmOptionsBase {
    * @default LESS_THAN_LOWER_OR_GREATER_THAN_UPPER_THRESHOLD
    */
   readonly comparisonOperator?: ComparisonOperator;
+
+  /**
+   * The period over which the anomaly detection band's statistics are applied.
+   * This period is passed into the underlying math expression.
+   *
+   * This period overrides the period set in the metric used.
+   *
+   * @default Duration.minutes(5)
+   */
+  readonly period?: cdk.Duration;
```

Comment:
I'm not sure if it's a bad pattern to add a property to a child interface when the same property exists on the parent, especially when the parent property is marked as deprecated. Maybe I could rename this to `anomalyDetectionPeriod` or something similar.